modules/dnn/src/libdnn/include/greentea_math_functions.hpp

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
@vpisarev , Thanks for your comments, we will check every single file to make sure it is clean and has proper license and copyright.